Document use of action outputs in readme (#1)
fregante/daily-version-action: README.md

Creates a new tag using the format `Y.M.D` (using [daily-version](https://github.com/fregante/daily-version)), but only if `HEAD` isn’t already tagged.

Ideally used on schedule, but you could also change the suggested condition to only make it run on `master`.

## Usage

It expects `git` to be configured to push to the same repo, perhaps via [fregante/setup-git-token](https://github.com/fregante/setup-git-token)

See [action.yml](action.yml)

```yaml
  Version:
    steps:
    - uses: actions/checkout@v2
    - uses: fregante/setup-git-token@v1 # Allows pushing the generated tag to the repo
      with:
        token: ${{ secrets.GITHUB_TOKEN }}
    - name: Create tag if necessary
      if: github.event_name == 'schedule' # Ensures it only runs on a schedule
      uses: fregante/daily-version-action@v1
```

> To check if any new version has been created in other jobs, use [`job.outputs`](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjobs_idoutputs) object to see if `created` property is set. This property would not be set if any new versions are created.

Reviewer: ```suggestion
You can use the `created` output of this action to test whether a new version has been created, even [across jobs](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjobs_idoutputs):
```

```yaml

>   Build:
>     needs: Version
>     if: github.event_name == 'push' || needs.Version.outputs.created

Reviewer: Let's keep it simple
```suggestion
    - name: 'Create tag if necessary'
      id: version
      uses: fregante/daily-version-action@v1
    - name: 'Created?'
      if: steps.version.outputs.created
      runs: echo 'Created!' ${{ steps.version.outputs.version }}
```

Untested though.

Author: How 'bout adding `steps` at the start. For verbosity.

Reviewer: Sure, but then we also need `setup-git-token`

```

> 
> Regradless of creating a new version, the most recent version can be accessed using the `job.outputs.version` property, after this action is run.
> 
> ``` yaml
>   Cleanup:
>     needs: Version
>     run: echo "Current version: ${{needs.Version.outputs.version}}"
> ```

Reviewer: This can be dropped for now since the previous example shows both outputs and now there's an **Outputs** section.

I plan on added a full example from https://github.com/notlmn/browser-extension-template/pull/28 once it's finished
```suggestion
```
